### PR TITLE
New Version 1.4

### PR DESCRIPTION
**Pull request by hirotomoyamada**
## Description

Update version 1.4

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/components/visually-hidden/tests/visually-hidden.test.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@yamada-ui/test"
+import { VisuallyHidden } from "../src"
+
+describe("<VisuallyHidden />", () => {
+  test("VisuallyHidden renders correctly", async () => {
+    render(<VisuallyHidden>Test</VisuallyHidden>)
+    const el = screen.getByText("Test")
+    expect(el.textContent).toBe("Test")
```
Comment (illionillion) [inline packages/components/visually-hidden/tests/visually-hidden.test.tsx:8]: Wouldn't `toBeInTheDocument()` or `toEqual()` be better?

```ts
expect(el.textContent).toEqual("Test")
// or
expect(el.textContent).toBeInTheDocument()
```

File: packages/components/visually-hidden/tests/visually-hidden.test.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  })
+
+  test("VisuallyHidden is visually hidden", async () => {
+    render(<VisuallyHidden>Test</VisuallyHidden>)
+    const style = window.getComputedStyle(screen.getByText("Test"))
```
Comment (illionillion) [inline packages/components/visually-hidden/tests/visually-hidden.test.tsx:13]: Wouldn't `toHaveStyle` be better?

https://github.com/testing-library/jest-dom?tab=readme-ov-file#tohavestyle

File: packages/components/visually-hidden/tests/visually-hidden.test.tsx (hunk 1 above)
Comment (hirotomoyamada) [reply packages/components/visually-hidden/tests/visually-hidden.test.tsx:8]: This PR is to raise the version and does not require a review. If there is an improvement, please create another issue and discuss it.

File: packages/components/visually-hidden/tests/visually-hidden.test.tsx (hunk 2 above)
Comment (hirotomoyamada) [reply packages/components/visually-hidden/tests/visually-hidden.test.tsx:13]: This PR is to raise the version and does not require a review. If there is an improvement, please create another issue and discuss it.

**Pull request merged**